examples/md-flexible/src/domainDecomposition/RegularGridDecomposition.cpp

```diff
@@ -39,6 +39,12 @@ RegularGridDecomposition::RegularGridDecomposition(const MDFlexConfig &configura
 
   _decomposition = DomainTools::generateDecomposition(_subdomainCount, configuration.subdivideDimension.value);
 
+  double maxSigma{0};
+  for (auto sigmaMapElement : configuration.sigmaMap.value) {
+    maxSigma = std::max(maxSigma, sigmaMapElement.second);
+  }
+  _maxReflectiveSkin = sixthRootOfTwo * maxSigma / 2.;
```

Comment:
add a comment explaining this calculation

Reply:
See latest commit